documentation/modules/ROOT/pages/operations/debezium-server.adoc

```diff
@@ -632,6 +632,28 @@ This value will be doubled upon every retry but won't exceed `debezium.sink.redi
 |`10000`
 |Max delay when encountering Redis connection or OOM issues.
 
+|[[redis-trim-maxlen]]<<redis-trim-maxlen, `debezium.sink.redis.trim.maxlen`>>
+|
+|(Optional) Evicts entries as long as the stream's length exceeds the specified threshold.
+
+|[[redis-trim-relative-minid]]<<redis-trim-relative-minid, `debezium.sink.redis.trim.relative.minid`>>
+|
+|(Optional) Specify the milliseconds, relative to the current time,
+for trimming the stream. For example, if you want to continuously
+evict all items older than 1 hour on each; use `3600000`.
+
+|[[redis-trim-limit]]<<redis-trim-limit, `debezium.sink.redis.trim.limit`>>
```

Comment:
Let's discuss this after the next one, so as to avoid the forward reference to `redis.trim.exact`.